Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env groovy
+
+devImageTag = "cfrdsl:dev"
+
+node('docker && awsaccess') {
+  cleanWorkspace()
+
+  stage('Checkout') {
+    checkout scm
+  }
+
+  stage("Build") {
+    image = docker.build(devImageTag, "-f Dockerfile.dev .")
+  }
+
+  image.inside {
+    stage('Run linter') {
+      // TODO: [DP-108] fix rubocop violations
+      sh 'bundle exec rubocop || true'
```

Comment:
If you use `--fail-level F` instead of `|| true`, rubocop will die at this stage if there are any fatal errors, but let anything else go through. It should pass with no issues at this stage though